kedro/framework/cli/hooks/manager.py

```diff
@@ -42,7 +42,7 @@ def _register_cli_hooks_setuptools(self) -> None:
         }
 
         if plugin_names:
-            logging.info(
+            logging.getLogger(__name__).info(
```

Comment:
This is the pattern used elsewhere in kedro where `self._logger` isn't available. It means the messages will appear to come from `kedro.framework.hooks.manager` rather than `root`.